src/js/rx/containers/Active.jsx

```diff
@@ -1,17 +1,72 @@
 import React from 'react';
 import { connect } from 'react-redux';
+import { browserHistory } from 'react-router';
 
 import PrescriptionList from '../components/PrescriptionList';
+import PrintList from '../components/PrintList';
+import SortMenu from '../components/SortMenu';
+
 import { loadData } from '../actions/prescriptions.js';
 
 class Active extends React.Component {
   componentWillMount() {
     this.props.dispatch(loadData());
+    this.handleSortOnChange = this.handleSortOnChange.bind(this);
+    this.handleSortOnClick = this.handleSortOnClick.bind(this);
+    this.dispatchSortAction = this.dispatchSortAction.bind(this);
+  }
+
+  dispatchSortAction(action) {
+    this.props.dispatch({ type: action });
+  }
+
+  handleSortOnChange(domEvent) {
+    if (domEvent.type === 'change') {
+      browserHistory.push({
+        pathname: '/rx',
+        query: {
+          sort: domEvent.target.value
+        }
+      });
+    }
+    this.dispatchSortAction(domEvent.target.value);
+  }
+
+  handleSortOnClick(domEvent) {
+    const fullURL = domEvent.target.href;
+
+    // Find the sort parameter, split the query string on the = and retrieve value
+    const sortParam = fullURL.match(/sort=[-a-z]{1,}/i)[0].split('=')[1];
```

Comment:
Discussion about this line was discarded from the last commit: https://github.com/department-of-veterans-affairs/vets-website/pull/2856/files/d7fee042f9ef69bc753becc7d85c7d5f8fc64045#diff-d35839005110b4b292b8334ed4bbd76bR39